docs/features/search/search-engines.md

```diff
@@ -237,4 +237,17 @@ search:
     batchSize: 100
 ```
 
-> You can also increase the batch size if you are using a large ES instance.
+### Elasticsearch query config
+
+By default the default settings for the Elasticsearch queries is used. If you need to tweak the fuzziness of the query results you can do this with 2 parameters, fuzziness and prefixLength.
```

Comment:
```suggestion
By default the default settings for the Elasticsearch queries is used. If you need to tweak the fuzziness of the query results you can do this with 2 parameters, `fuzziness` and `prefixLength`.
```

That should fix the Vale error. I also added them to fuzziness as it's a parameter so helps the reader know it's a code concept 👍